src/main/cli/commandLine.ts

```diff
@@ -0,0 +1,120 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { readerActions } from "readium-desktop/common/redux/actions";
+import { container } from "readium-desktop/main/di";
+import { extractHostname } from "readium-desktop/utils/hostname";
+import { Store } from "redux";
+import * as yargs from "yargs";
+
+import { OpdsFeedRepository } from "readium-desktop/main/db/repository/opds";
+import { PublicationRepository } from "readium-desktop/main/db/repository/publication";
+import { RootState } from "readium-desktop/main/redux/states";
+import { CatalogService } from "readium-desktop/main/services/catalog";
+
+export interface ICliParam {
+    readonly argv: yargs.Arguments;
+    quit: boolean;
+}
+
+export interface ICli {
+    name: string;
+    fct: (o: ICliParam) => Promise<any>;
+    help: string[];
+}
+
+export const cli: ICli[] = [
+    {
+        name: "_",
+        fct: async ({ argv }) => {
+            let notRead = true;
+
+            argv._.map(async (path) => {
```

Comment:
Strictly-speaking, `path` is not a reserved keyword, but it is a commonly-used NodeJS API (e.g. `path.join()`). I would therefore recommend using a different name for this local variable, for the avoidance of doubt.